binutils/binutils/configure

```diff
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /data/data/com.termux/files/usr/bin/sh
```

Comment:
Your build tools keep doing this. Whatever it is that you're doing, you should do it on a computer, not on Termux...